credits/models.py

```diff
@@ -15,7 +19,8 @@ class Profile(models.Model):
 
 
 def user_post_save(sender, instance, created, raw, **kwargs):
-    if created:
+    # Use this table only in testing
+    if created and TESTING:
```

Comment:
Can we make a small test for this?

Reply:
I don't know how can we test this, ideas?

Reply:
I don't know how to change the `sys.argv` in tests, maybe we can just create a `User` and ensure the `User` created is indeed a `Profile`, does it make any sense?

Anyway, approved ✅

Reply:
Definition of `sys` in python: This module provides access to some variables used or maintained by the interpreter and to functions that interact strongly with the interpreter. It is always available.

When you execute the `python manage.py test` command, sys would have the test argument in sys.argv.
So, in testing it always be executed, and I don't know if it's possible to modify that without problems